src/Illuminate/Database/Schema/Builder.php

```diff
@@ -67,7 +67,29 @@ public function hasColumn($table, $column)
 		return in_array($column, array_map('strtolower', $this->getColumnListing($table)));
 	}
 
-	/**
+    	/**
+     	* Determine if the given table has given columns.
+     	*
+     	* @param  string  $table
+     	* @param  array   $columns
+     	* @return bool
+     	*/
+    	public function hasColumns($table, $columns)
+    	{
+        	$tableColumns = $this->getColumnListing($table);
+        	
+        	array_map('strtolower', $tableColumns);
+
+        	foreach ($columns as $column)
+        	{
+            		if ( ! in_array(strtolower($column), $tableColumns))
+                		return false;
```

Comment:
no. this MUST go on the same line, or you MUST use braces. Code without that is asking for trouble.